forum/tests.py

```diff
@@ -13,63 +26,305 @@
+        self.assertEquals(response.status_code, status.HTTP_204_NO_CONTENT)
+        with self.assertRaises(Transcript.DoesNotExist):
+            Transcript.objects.get(semester=self.semester, owner=self.student)
+
+    def test_create_comments(self):
+        setUpTranscript(self)
+        content = 'Stan Oh My Girl, fromis_9, f(x), LOONA'
+        data = {
+            'content': content,
+            'timestamp': datetime.datetime.now(),
+            'jhed': self.student.jhed,
+        }
+        request = self.factory.post(
+            '/advising/forum/Fall/2019/', data=data, format='json')
+        response = get_response_for_semester(self, request, self.student.user)
+        self.assertEquals(response.status_code, status.HTTP_201_CREATED)
+        comment = Comment.objects.get(
+            transcript=self.transcript, author=self.student)
+        self.assertEquals(content, comment.content)
+
+        content = 'Stan YUKIKA - literally Simon'
+        data = {
+            'content': content,
+            'timestamp': datetime.datetime.now(),
+            'jhed': self.student.jhed,
+        }
+        request = self.factory.post(
+            '/advising/forum/Fall/2019/', data=data, format='json')
+        response = get_response_for_semester(self, request, self.advisor.user)
+        self.assertEquals(response.status_code, status.HTTP_201_CREATED)
+        comment = Comment.objects.get(
+            transcript=self.transcript, author=self.advisor)
+        self.assertEquals(content, comment.content)
+
+    def test_add_advisor(self):
+        setUpTranscriptNoAdvisor(self)
+        self.assertEquals(self.transcript.advisors.count(), 0)
+        user = User.objects.create_user(
+            username='rbiz',
+            password='k',
+            first_name='Rishi',
+            last_name='Biswas',)
+        advisor = Student.objects.create(user=user)
+        advisor.jhed = 'rbiswas4'
+        advisor.save()
+        data = {
+            'action': 'add',
+            'jhed': advisor.jhed,
+        }
+        request = self.factory.patch(
+            '/advising/forum/Fall/2019/', data=data, format='json')
+        response = get_response_for_semester(self, request, self.student.user)
+        self.assertEquals(response.status_code, status.HTTP_200_OK)
+        self.assertEquals(self.transcript.advisors.count(), 1)
+
+    def test_add_advisor_not_in_db(self):
+        setUpTranscriptNoAdvisor(self)
+        self.assertEquals(self.transcript.advisors.count(), 0)
+        data = {
+            'action': 'add',
+            'jhed': 'rbiswas4',
+        }
+        request = self.factory.patch(
+            '/advising/forum/Fall/2019/', data=data, format='json')
+        response = get_response_for_semester(self, request, self.student.user)
+        self.assertEquals(response.status_code, status.HTTP_404_NOT_FOUND)
+        self.assertEquals(self.transcript.advisors.count(), 0)
+
+    def test_add_advisor_already_exists(self):
```

Comment:
I would also change this to `test_add_added_advisor`
(Let's use the word "exists" for "in the database" and "added" for "on the list of advisors in the transcript"